vault/logical_system.go

```diff
@@ -1717,6 +1733,118 @@ func (b *SystemBackend) handleTuneWriteCommon(ctx context.Context, path string,
 		}
 	}
 
+	// user-lockout config
+	{
+		var apiuserLockoutConfig APIUserLockoutConfig
+
+		userLockoutConfigMap := data.Get("user_lockout_config").(map[string]interface{})
```

Comment:
Could this assertion panic if `user_lockout_config` is not passed in?

Reply:
I followed the same way we check for config field in handleMount. I tried tests without passing user_lockout_config and it does not panic.